providers/fips/self_test_data.inc

```diff
@@ -191,3 +213,176 @@ static const ST_KAT_KDF st_kat_kdf_tests[] =
     }
 };
 
+/*-
+* DRBG test vectors are a small subset of
+* https://csrc.nist.rip/groups/STM/cavp/documents/drbg/drbgtestvectors.zip
+* Using the folder drbgvectors_pr_true
+* Generated for CAVS 14.3.
+*/
+
+static const unsigned char drbg_hash_sha256_pr_entropyin[] = {
+    0x06, 0x6d, 0xc8, 0xce, 0x75, 0xb2, 0x89, 0x66, 0xa6, 0x85, 0x16, 0x3f,
+    0xe2, 0xa4, 0xd4, 0x27, 0xfb, 0xdb, 0x61, 0x66, 0x50, 0x61, 0x6b, 0xa2,
+    0x82, 0xfc, 0x33, 0x2b, 0x4e, 0x6f, 0x12, 0x20
+};
+static const unsigned char drbg_hash_sha256_pr_nonce[] = {
+    0x55, 0x9f, 0x7c, 0x64, 0x89, 0x70, 0x83, 0xec, 0x2d, 0x73, 0x70, 0xd9,
+    0xf0, 0xe5, 0x07, 0x1f
+};
+static const unsigned char drbg_hash_sha256_pr_persstr[] = {
+    0x88, 0x6f, 0x54, 0x9a, 0xad, 0x1a, 0xc6, 0x3d, 0x18, 0xcb, 0xcc, 0x66,
+    0x85, 0xda, 0xa2, 0xc2, 0xf7, 0x9e, 0xb0, 0x89, 0x4c, 0xb4, 0xae, 0xf1,
+    0xac, 0x54, 0x4f, 0xce, 0x57, 0xf1, 0x5e, 0x11
+};
+static const unsigned char drbg_hash_sha256_pr_entropyinpr0[] = {
+    0xff, 0x80, 0xb7, 0xd2, 0x6a, 0x05, 0xbc, 0x8a, 0x7a, 0xbe, 0x53, 0x28,
+    0x6b, 0x0e, 0xeb, 0x73, 0x3b, 0x71, 0x5a, 0x20, 0x5b, 0xfa, 0x4f, 0xf6,
+    0x37, 0x03, 0xde, 0xad, 0xb6, 0xea, 0x0e, 0xf4
+};
+static const unsigned char drbg_hash_sha256_pr_entropyinpr1[] = {
+    0xc7, 0x38, 0x32, 0x53, 0x46, 0x81, 0xed, 0xe3, 0x7e, 0x03, 0x84, 0x6d,
+    0x3c, 0x84, 0x17, 0x67, 0x29, 0x7d, 0x24, 0x6c, 0x68, 0x92, 0x41, 0xd2,
+    0xe7, 0x75, 0xbe, 0x7e, 0xc9, 0x96, 0x29, 0x3d
+};
+static const unsigned char drbg_hash_sha256_pr_addin0[] = {
+    0xb7, 0x21, 0x5f, 0x14, 0xac, 0x7b, 0xaf, 0xd0, 0xa9, 0x17, 0x72, 0xba,
+    0x22, 0xf7, 0x19, 0xaf, 0xbd, 0x20, 0xb3, 0x11, 0x63, 0x6c, 0x2b, 0x1e,
+    0x83, 0xe4, 0xa8, 0x23, 0x35, 0x3f, 0xc6, 0xea
+};
+static const unsigned char drbg_hash_sha256_pr_addin1[] = {
+    0xce, 0xd3, 0x1f, 0x7e, 0x0d, 0xae, 0x5b, 0xb5, 0xc0, 0x43, 0xe2, 0x46,
+    0xb2, 0x94, 0x73, 0xe2, 0xfd, 0x39, 0x51, 0x2e, 0xad, 0x45, 0x69, 0xee,
+    0xe3, 0xe3, 0x80, 0x33, 0x14, 0xab, 0xa7, 0xa3
+};
+static const unsigned char drbg_hash_sha256_pr_expected[] = {
+    0x60, 0xc2, 0x34, 0xcf, 0xaf, 0xb4, 0x68, 0x03, 0x3b, 0xf1, 0x95, 0xe5,
+    0x78, 0xce, 0x26, 0x6e, 0x14, 0x65, 0x32, 0x6a, 0x96, 0xa9, 0xe0, 0x3f,
+    0x8b, 0x89, 0x36, 0x70, 0xef, 0x62, 0x75, 0x4d, 0x5e, 0x80, 0xd5, 0x53,
+    0xa1, 0xf8, 0x49, 0x50, 0x20, 0x8b, 0x93, 0x43, 0x07, 0x9f, 0x2e, 0xf8,
+    0x56, 0xe9, 0xc5, 0x70, 0x61, 0x85, 0x97, 0xb5, 0xdc, 0x82, 0xa2, 0xda,
+    0xea, 0xa3, 0xfd, 0x9b, 0x2f, 0xd2, 0xa0, 0xd7, 0x1b, 0xc6, 0x29, 0x35,
+    0xcc, 0xb8, 0x3d, 0xa0, 0x67, 0x98, 0x05, 0xa0, 0xe3, 0x1e, 0xfe, 0xe4,
+    0xf0, 0xe5, 0x13, 0xb0, 0x83, 0x17, 0xfa, 0xca, 0x93, 0x5e, 0x38, 0x29,
+    0x48, 0xd2, 0x72, 0xdb, 0x76, 0x3e, 0x6d, 0xf3, 0x25, 0x10, 0xff, 0x1b,
+    0x99, 0xff, 0xf8, 0xc6, 0x0e, 0xb0, 0xdd, 0x29, 0x2e, 0xbc, 0xbb, 0xc8,
+    0x0a, 0x01, 0x6e, 0xd3, 0xb0, 0x0e, 0x4e, 0xab
+};
+
+/*-                                                                       */
```

Comment:
```suggestion
/*
 * CTR_DRBG.rsp
 * 
 * [AES-128 use df]
 * [PredictionResistance = True]
 * [EntropyInputLen = 128]
 * [NonceLen = 64]
 * [PersonalizationStringLen = 128]
 * [AdditionalInputLen = 128]
 * [ReturnedBitsLen = 512]
 * 
 * COUNT = 0
 * EntropyInput = 92898f31fa1cff6d182f260643dff818
 * Nonce = c2a4d972c3b9b697
 * PersonalizationString = ea65ee60264e7eb60e8268c4373c5c0b
 * AdditionalInput = 1a40fae3cc6c7ca0f8daba59236dad1d
 * EntropyInputPR = 20728a06f86f8dd441e272b7c42ce810
 * AdditionalInput = 9f72766cc746e5ed2e532012bc59318c
 * EntropyInputPR = 3db0f094f305503317863e2208f7a501
 * ReturnedBits = 5a3539870f4d22a40924ee71c96fac720ad6f08882d0832873ec3f93d8ab4523
 *                f07eac45145e939fb1d676433db6e80888f6da89087742fe1af43fc423c51f68
 */
```

Reply:
(I removed and readded this suggestion because multiline-suggestions are not supported by GitHub yet.)